.github/workflows/scripts/e2e/gmc_test.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -xe
+LOG_PATH=/home/$(whoami)/logs
+IMAGE_REPO=${OPEA_IMAGE_REPO:-docker.io}
+MOUNT_DIR=/home/$USER_ID/charts-mnt
+
+function validate_chatqna() {
+
+   init_chatqna
+
+   kubectl create ns system
```

Comment:
Please use a random suffix in your namespace. Because there will be several tests running in parallel in a same K8s cluster. Fixed namespace will cause conflict.

Reply:
I will suggest we use opea-system instead of system here.